digitalstrategie/assets/scss/style.scss

```diff
@@ -58,3 +58,4 @@
 @import 'components/footer';
 @import 'components/header';
 @import 'components/books';
+@import 'components/forms';
```

Comment:
this is a total nit pick so sorry about that but as component files should reflect the block class name to which they refer I would also call the file 'form' not 'forms'